src/Sarif/Core/Notification.cs

```diff
@@ -1,7 +1,9 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+#if DEBUG
```

Comment:
>DEBUG [](http://example.com/codeflow?start=4&length=5)

Please note this distinction you may encounter, I typically build as DEBUG in the IDE, our command-line is release. And so you may see discrepancies in style issues between IDE and command-line builds.